packages/tools/devtools/devtools-example/src/index.tsx

```diff
@@ -29,6 +29,9 @@ ReactDOM.render(<DevtoolsView />, devtoolsElement, () => {
 	console.log("Devtools UI rendered!");
 });
 
+// Setting "fluidStarted" is just for our test automation
+globalThis.fluidStarted = true;
```

Comment:
Should this maybe go in the callback to the `ReactDOM.render()` call that renders the app? So it's set once we really have everything in the DOM. In here, this could technically get set to `true` before the app is visible.

Reply:
Resolved